src/puck/components/Header.tsx

```diff
@@ -195,16 +196,10 @@ const ToggleEntityFields = () => {
     <TooltipProvider>
       <Tooltip>
         <TooltipTrigger asChild>
-          <Button
-            variant="ghost"
-            size="icon"
-            onClick={toggleTooltips}
-            className={
-              tooltipsVisible ? "border-2 border-[#5A58F2] rounded-full" : ""
-            }
-          >
-            <MessageSquareText className="sm-icon" />
-          </Button>
+          <div className="flex flex-row self-center gap-3 pl-2">
```

Comment:
Per the item the "Show Entity Fields" should be in a pill and only show when the toggle is enabled.

Don't make the toggle have an icon in it. There's been a thread about not doing that. I'll update the item description to remove that.

Reply:
This all needs to move to visual-editor lib since Brian is moving those things. It's all just waiting on a semgrep approval